src/utils/api/helper.ts

```diff
@@ -592,16 +591,8 @@ export const calculateFees = async ({
 export const proceedToPayment = async (
   {
     transaction,
-    cardData,
   }: {
     transaction: NewTransactionResponse;
-    cardData: {
-      brand: string;
-      cvv: string;
-      pan: string;
-      expiryDate: string;
-      holderName: string;
-    };
   },
```

Comment:
Here, I think it's better to have just one 'transaction' param instead of an object with only one 'transaction' property.